### PR TITLE
Add badge in menu and rewards count

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: src/App/Layout/Header/View.tsx [hunk 1]
```diff
@@ -35,11 +37,28 @@ export default function HeaderView({ menuItems, impersonating = false }: HeaderV
...
   const hideHeader = (isMatchProjectDetail || isMatchSettings) && !isXl;
 
+  const rewardSum = useMemo(
+    () => billingProfile?.billingProfiles?.reduce((acc, profile) => acc + profile.invoiceableRewardCount, 0),
+    [billingProfile]
+  );
+
+  const rewardBadgeContent = useMemo(() => {
+    if (!rewardSum) {
+      return undefined;
```
Comment (haydencleary) [inline src/App/Layout/Header/View.tsx:52]:
```suggestion
      return null;
```

File: src/App/Layout/Header/View.tsx [hunk 2]
```diff
@@ -35,11 +37,28 @@ export default function HeaderView({ menuItems, impersonating = false }: HeaderV
...
+    }
+
+    if (rewardSum > 9) {
+      return `+9`;
```
Comment (haydencleary) [inline src/App/Layout/Header/View.tsx:56]:
```suggestion
      return "+9";
```

File: src/App/Layout/Header/View.tsx [hunk 3]
```diff
@@ -35,11 +37,28 @@ export default function HeaderView({ menuItems, impersonating = false }: HeaderV
...
+    }
+
+    return `${rewardSum}`;
```
Comment (haydencleary) [inline src/App/Layout/Header/View.tsx:59]:
```suggestion
    return rewardSum;
```

**Review by haydencleary — CHANGES_REQUESTED** [review]

File: src/App/Layout/Header/View.tsx (hunk 1 above)
Comment (alexbeno) [reply src/App/Layout/Header/View.tsx:52]: should be undefined for the next ui component

File: src/App/Layout/Header/View.tsx (hunk 3 above)
Comment (alexbeno) [reply src/App/Layout/Header/View.tsx:59]: content should be a string

**Review by haydencleary — APPROVED** [review]

**Pull request merged**